interfaces/builtin/alsa.go

```diff
@@ -45,6 +45,19 @@ const alsaConnectedPlugAppArmor = `
 @{PROC}/asound/** rw,
 `
 
+const alsaConnectedPlugUdev = `
+# This file contains udev rules for alsa devices.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="controlC[0-9]*", NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="hw[CD0-9]*",     NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="pcm[CD0-9cp]*",  NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="midiC[D0-9]*",   NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="timer",          NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="seq",            NAME="snd/%%k", TAG+="%[1]s"
```

Comment:
Adding the name here ensures that the sound device node appears exclusively in the `/dev/snd` directory where they are created in my system and also it meant to be consistent with  `/dev/snd/*` in apparmor policy for alsa interface. Removing the name results in sound devices are created under `/dev` directory by default. But I couldn't find them there(`/dev`) in my system.

Basically, it's harmless to have 'NAME' here for each device. I need to confirm this at this point before I proceed.

`SUBSYSTEM=="sound", KERNEL=="card[0-9]*", TAG+="%[1]s"`
Yes, I saw it. Thanks for the mention.